### PR TITLE
DW-1044-perform-test-forms

### PR DESCRIPTION
**Pull request by silsanchez8**
## Modifications:

* fix(testForms): perform unit form tests, using this advices: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen

### CODE REVIEW

**Comment by andresmoschini** [comment]
❤️

File: src/components/DopplerPlus/UpgradeSuggestionForm.test.js [hunk 1]
```diff
@@ -45,165 +45,160 @@ describe('UpgradeSuggestionForm component', () => {
...
     // Assert
     await waitFor(() => {
-      return expect(getByText('validation_messages.error_required_field')).toBeInTheDocument();
+      return expect(
+        screen.getByText('validation_messages.error_required_field'),
+      ).toBeInTheDocument();
     });
   });
 
   it('should show error message if name field is empty', async () => {
```
Comment (cbernat) [inline src/components/DopplerPlus/UpgradeSuggestionForm.test.js:124]: This test shouldn't be here anymore right? It is replaced by `'should show error message if email field is empty'`...

**Comment by jcamposmk** [comment]
:+1:

File: src/components/DopplerPlus/UpgradeSuggestionForm.test.js (hunk 1 above)
Comment (cbernat) [reply src/components/DopplerPlus/UpgradeSuggestionForm.test.js:124]: But this changes could be done in another PR so this is not an infinite PR.

**Comment by cbernat** [comment]
👍

**Comment by cbernat** [comment]
🚀 ready to ship!.

**Pull request merged**